### DISCUSSION TITLE
Can I easily put something in a grid?

### DISCUSSION THREAD

**Discussion by andrewducker** [EvotecIT/PSWriteHTML#461] [Q&A] [answered]
Basically, I want some additional details in my dashboard, at the top of the first tab, saying what department it was generated for, and when it was generated.  And I just want those aligned above each other.
And I could just stuff both of those into a table, and then use Table -DataTable, but although that allows me to disable a bunch of stuff, it doesn't allow me to disable the header.
So is there an alternative that would allow me to line things up nicely, but not put a header in?

```
Dashboard -Name "Status" -FilePath d:\status.html {
	Tab -Name "Overview"{
		Section -Name "Generation Details"{
			$overview = @()
			$overview += [pscustomobject]@{
				DataType = "Department"
				DataContent = "My Department"
			}
			$overview += [pscustomobject]@{
				DataType = "Generated at"
				DataContent = Get-Date
			}
			Table -DataTable $overview -DisablePaging -DisableOrdering -DisableInfo -HideFooter -HideButtons -DisableSearch -DisableResponsiveTable
		}
	}
}
```

**Comment by PrzemyslawKlys** [comment]
Why do you want it in a table? it seems to me like there are multiple ways to show it?

![image](https://github.com/user-attachments/assets/952c4b52-482e-46ba-bbb1-ca4fc804decb)

**Comment by andrewducker** [comment]
I'd be delighted to display it like that at the top of the page!  Can you share the code that did that?

**Reply by PrzemyslawKlys** [reply, marked as answer]
```powershell
  New-HTMLHeader {
                    New-HTMLSection -Invisible {
                        New-HTMLSection {
                            New-HTMLText -Text "Report generated on $(Get-Date)" -Color Blue
                        } -JustifyContent flex-start -Invisible
                        New-HTMLSection {
                            New-HTMLText -Text "ADEssentials - $($Script:Reporting['Version'])" -Color Blue
                        } -JustifyContent flex-end -Invisible
                    }
                }
```

**Reply by andrewducker** [reply]
Thank you!
